assets/dev/js/editor/component.js

```diff
@@ -0,0 +1,14 @@
+import ComponentBase from 'elementor-api/modules/component-base';
+import EditorDocumentsComponent from './components/documents/component';
+
+export default class Component extends ComponentBase {
```

Comment:
Why do you need that?

Reply:
Will be used for export `container-helper` to be externals.

Reply:
Please explain in code or in private

Reply:
As i said, the container-helper will be exported via this component, since the nested is a compiled module.